### ISSUE TITLE
Harbor Replication  doesn't pulling openshift installation images

### ISSUE THREAD

**Issue by Edenzil** [goharbor/harbor#17421]
We have images for the installation of openshift 4.8.19  jfrog artifactory ( we mirrored it) . 

And we need to replicate them to our local harbor registry 2.4.2. 
I can replicate all the other images but those not. 

I will really appreciate your help

**Comment by chlins** [comment]
Could you provide more information such as replication policy, logs?

**Comment by Edenzil** [comment]
Yes , there are no logs , it says that there are no resources to be found

**Comment by Edenzil** [comment]
Also if I push a "regular image" to the same repo where the installation of openshift are , harbor does pull that image but not the installation ones.

**Comment by chlins** [comment]
Could you provide the screenshot of your replication rule?

**Comment by Edenzil** [comment]
yes here is the rule
![WhatsApp Image 2022-08-24 at 11 35 39 AM](https://user-images.githubusercontent.com/91659053/186395139-6ed3c1c4-31f6-4592-9ef2-6487112cdeb7.jpeg)
